src/biconnectivity/biconnect.jl

```diff
@@ -22,7 +23,9 @@ end
 
 Perform a DFS visit storing the depth and low-points of each vertex.
 """
-function visit!(g::AbstractGraph, state::Biconnections, u::Integer, v::Integer)
+function visit!(g::AbstractGraph, state::Biconnections{E}, u::Integer, v::Integer) where {E}
+    @assert E === Edge{eltype(g)}
```

Comment:
Let's not use `assert` - it's really just for development. If we want to fail if `E !== Edge{eltype(g)}`, let's test and throw an appropriate error.

Reply:
Can I simply put this into a comment? The `assert` should not be triggered anyways, it is just there for informing a reader that `E === Edge{eltype(g)}`.

Reply:
That works.